js/cli/src/namespace.ts

```diff
@@ -200,15 +214,48 @@ CLI.programCommandWithConfig(
   "uncache_artifact",
   async (config, options, _files) => {
     const { keypair, env, rpcUrl } = options;
-    const namespaceProgram = await NamespaceProgram.getProgramWithWalletKeyPair(
-      NamespaceProgram,
-      await Wallet.loadWalletKey(keypair),
-      env,
-      rpcUrl
-    );
+
+    const namespaceProgram = await initNsProgram(rpcUrl, keypair);
+
+    const accounts: nsIx.UncacheArtifactAccounts = {
+      namespaceMint: new web3.PublicKey(config.mint),
+      artifact: config.artifact,
+    };
+
+    const args: nsIx.UncacheArtifactArgs = {
+      page: new anchor.BN(config.page),
+    };
+
+    await namespaceProgram.uncacheArtifact(args, accounts);
 
     log.info("Artifact removed from Namespace cache");
   }
 );
 
 CLI.Program.parseAsync(process.argv);
+
+async function initNsProgram(
```

Comment:
This is missing the `env` argument convention that can be passed in to all the other cli commands.

Reply:
I'm not sure what the `env` argument would do in this function, the `rpcUrl` already takes care of which env you would be pointed to.

Reply:
Also this is an internal function used by the higher up cli commands so it shouldn't break any user flow